dist/index.js

```diff
@@ -7086,6 +7089,115 @@ class MapboxPathControl {
             ],
         };
     }
+    setLineString(feature) {
+        var _a, _b;
```

Comment:
If think that even in typescript, `let` should be prefered as `var` relative to is scope ?